salesforce-connector/processes/QueryService.p.json

```diff
@@ -0,0 +1,64 @@
+{
+  "format" : "10.0.0",
+  "id" : "18C0022FCA6F8CC0",
+  "kind" : "CALLABLE_SUB",
+  "config" : {
+    "data" : "com.axonivy.connector.salesforce.connector.QueryServiceData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "CallSubStart",
+      "name" : "call(String)",
+      "config" : {
+        "callSignature" : "call",
+        "input" : {
+          "params" : [
+            { "name" : "query", "type" : "String" }
+          ],
+          "map" : {
+            "out.q" : "param.query"
+          }
+        },
+        "result" : {
+          "params" : [
+            { "name" : "jsonNode", "type" : "com.fasterxml.jackson.databind.JsonNode" }
+          ],
+          "map" : {
+            "result.jsonNode" : "in.jsonNode"
+          }
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 64 }
+      },
+      "connect" : { "id" : "f4", "to" : "f3" }
+    }, {
+      "id" : "f1",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 416, "y" : 64 }
+      }
+    }, {
+      "id" : "f3",
+      "type" : "RestClientCall",
+      "name" : "Query API",
+      "config" : {
+        "path" : "/query",
+        "clientId" : "a8bd1ef0-7283-4211-9672-3231654ad15f",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "q" : "in.q"
+        },
+        "statusErrorCode" : "ivy:error:rest:client",
+        "templateParams" : {
+          "subdomain" : ""
+        },
+        "resultType" : "com.fasterxml.jackson.databind.JsonNode",
+        "responseCode" : "in.jsonNode = result;"
```

Comment:
could you change this code to like this screenshot? Because we do not modify anything on result object
![image](https://github.com/axonivy-market/salesforce-connector/assets/83745591/08aaf3df-37bd-4117-a414-e8de8e1f0b6f)
@HoDacVu90